crates/pathfinder/src/rpc/v02.rs

```diff
@@ -236,6 +236,7 @@ pub fn register_all_methods(module: &mut jsonrpsee::RpcModule<RpcContext>) -> an
         method::estimate_fee::estimate_fee,
     )?;
     register_method(module, "starknet_getNonce", method::get_nonce::get_nonce)?;
+    register_method(module, "pathfinder_getProof", method::get_proof::get_proof)?;
```

Comment:
Methods that are neither in 0.1 nor in 0.2 Starkware specs are available on [yet another path](https://github.com/pscott/pathfinder/blob/995826e0957ecdc128948d3b97f4adfedbd49d43/crates/pathfinder/src/rpc.rs#L88).

Please register the method [here instead](https://github.com/pscott/pathfinder/blob/pathfinder_getProof/crates/pathfinder/src/rpc/pathfinder.rs).

Reply:
Ah yes this wasn't actually available when this work first started 😮‍💨

Reply:
Its unfortunately not quite that simple as the pathfinder endpoint has no context or framework yet.

I suggest we leave this as is in this PR, and then we patch this up afterwards once we have something in place?

Reply:
a8f1449f961b855b645bbbc958a235807c6e4bb8